code/style.css

```diff
@@ -138,3 +138,16 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
```

Comment:
Text appears outside the text box at one stage. Fix it by set a media query in mobile view or by adding flex-shrink in css for example.